### ISSUE TITLE
Conversion Failed

### ISSUE THREAD

**Issue by gibranlp** [weebney/webcamize#5]
**Describe the bug**
Im trying to run webcamize with a Canon EOS 600D and im getting an error 

**Terminal Output**
```console
/usr/local/bin/webcamize: Starting Canon EOS 600D on /dev/video0
/usr/local/bin/webcamize: Error executing gphoto2/ffmpeg
/usr/local/bin/webcamize: line 163: [: missing `]'

ffmpeg: ffmpeg version n6.0 Copyright (c) 2000-2023 the FFmpeg developers
  built with gcc 13.1.1 (GCC) 20230429
  configuration: --prefix=/usr --disable-debug --disable-static --disable-stripping --enable-amf --enable-avisynth --enable-cuda-llvm --enable-lto --enable-fontconfig --enable-gmp --enable-gnutls --enable-gpl --enable-ladspa --enable-libaom --enable-libass --enable-libbluray --enable-libbs2b --enable-libdav1d --enable-libdrm --enable-libfreetype --enable-libfribidi --enable-libgsm --enable-libiec61883 --enable-libjack --enable-libjxl --enable-libmfx --enable-libmodplug --enable-libmp3lame --enable-libopencore_amrnb --enable-libopencore_amrwb --enable-libopenjpeg --enable-libopenmpt --enable-libopus --enable-libpulse --enable-librav1e --enable-librsvg --enable-libsoxr --enable-libspeex --enable-libsrt --enable-libssh --enable-libsvtav1 --enable-libtheora --enable-libv4l2 --enable-libvidstab --enable-libvmaf --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxcb --enable-libxml2 --enable-libxvid --enable-libzimg --enable-nvdec --enable-nvenc --enable-opencl --enable-opengl --enable-shared --enable-version3 --enable-vulkan
  libavutil      58.  2.100 / 58.  2.100
  libavcodec     60.  3.100 / 60.  3.100
  libavformat    60.  3.100 / 60.  3.100
  libavdevice    60.  1.100 / 60.  1.100
  libavfilter     9.  3.100 /  9.  3.100
  libswscale      7.  1.100 /  7.  1.100
  libswresample   4. 10.100 /  4. 10.100
  libpostproc    57.  1.100 / 57.  1.100
[mjpeg @ 0x55db94534ec0] Format mjpeg detected only with low score of 25, misdetection possible!
Input #0, mjpeg, from 'fd:':
  Duration: N/A, bitrate: N/A
  Stream #0:0: Video: mjpeg (Baseline), yuvj422p(pc, bt470bg/unknown/unknown), 1056x704, 25 tbr, 1200k tbn
Stream mapping:
  Stream #0:0 -> #0:0 (mjpeg (native) -> rawvideo (native))
[swscaler @ 0x55db945b2740] deprecated pixel format used, make sure you did set range correctly
    Last message repeated 3 times
[video4linux2,v4l2 @ 0x55db94537240] ioctl(VIDIOC_G_FMT): Invalid argument
[out#0/video4linux2,v4l2 @ 0x55db94537e40] Could not write header (incorrect codec parameters ?): Invalid argument
[vost#0:0/rawvideo @ 0x55db94539380] Error initializing output stream:
Conversion failed!
/usr/local/bin/webcamize: line 16: kill: (31177) - No such process
```

**Expected behavior**
Run webcamize without errors 

**Desktop (please complete the following information):**
 - Distro and Version: Archlinux Kernel 6.4.8
 - Camera/Device: Canon EOS 600D
 - Webcamize version 1.1.1

**gphoto2 Information**
- Output from `gphoto2 --summary`
```console
Camera summary:
Manufacturer: Canon Inc.
Model: Canon EOS REBEL T3i
  Version: 3-1.0.2
  Serial Number: 74a86af14e804f829a112daecdcad660
Vendor Extension ID: 0xb (2.0)
Vendor Extension Description:

Capture Formats: JPEG
Display Formats: Association/Directory, Script, DPOF, MS AVI, MS Wave, JPEG, CRW, Unknown(b103), Unknown(bf02), Defined Type, Unknown(b104), Unknown(b105)

Device Capabilities:
	File Download, File Deletion, File Upload
	No Image Capture, No Open Capture, Canon EOS Capture, Canon EOS Capture 2

Storage Devices Summary:
store_00020001:
	StorageDescription: SD
	VolumeLabel:
	Storage Type: Removable RAM (memory card)
	Filesystemtype: Digital Camera Layout (DCIM)
	Access Capability: Read-Write
	Maximum Capability: 15923150848 (15185 MB)
	Free Space (Bytes): 11251023872 (10729 MB)
	Free Space (Images): -1

Device Property Summary:
Property 0xd402:(read only) (type=0xffff) 'Canon EOS REBEL T3i'
Property 0xd407:(read only) (type=0x6) 1
Property 0xd406:(readwrite) (type=0xffff) 'Unknown Initiator'
Property 0xd303:(read only) (type=0x2) 1
Battery Level(0x5001):(read only) (type=0x2) Enumeration [100,0,75,0,50] value: 100% (100)
```

- Output from `gphoto2 --autodetect`
```console
Model                          Port
----------------------------------------------------------
Canon EOS 600D                 usb:007,005
```

**Additional context**
Thats all

**Comment by weebney** [comment]
Please upgrade to Webcamize 1.1.2 and let me know if anything changes.

**Comment by gibranlp** [comment]
i have updated to 1.1.2

`console
webcamize: 1.1.2
`

And i get the same thing!

`console
webcamize: Starting Canon EOS 600D on /dev/video0
webcamize: Successfully started!
webcamize: Error executing gphoto2/ffmpeg

gphoto2: Capturing preview frames as movie to 'stdout'. Press Ctrl-C to abort.

ffmpeg: ffmpeg version n6.0 Copyright (c) 2000-2023 the FFmpeg developers
  built with gcc 13.1.1 (GCC) 20230429
  configuration: --prefix=/usr --disable-debug --disable-static --disable-stripping --enable-amf --enable-avisynth --enable-cuda-llvm --enable-lto --enable-fontconfig --enable-gmp --enable-gnutls --enable-gpl --enable-ladspa --enable-libaom --enable-libass --enable-libbluray --enable-libbs2b --enable-libdav1d --enable-libdrm --enable-libfreetype --enable-libfribidi --enable-libgsm --enable-libiec61883 --enable-libjack --enable-libjxl --enable-libmfx --enable-libmodplug --enable-libmp3lame --enable-libopencore_amrnb --enable-libopencore_amrwb --enable-libopenjpeg --enable-libopenmpt --enable-libopus --enable-libpulse --enable-librav1e --enable-librsvg --enable-libsoxr --enable-libspeex --enable-libsrt --enable-libssh --enable-libsvtav1 --enable-libtheora --enable-libv4l2 --enable-libvidstab --enable-libvmaf --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxcb --enable-libxml2 --enable-libxvid --enable-libzimg --enable-nvdec --enable-nvenc --enable-opencl --enable-opengl --enable-shared --enable-version3 --enable-vulkan
  libavutil      58.  2.100 / 58.  2.100
  libavcodec     60.  3.100 / 60.  3.100
  libavformat    60.  3.100 / 60.  3.100
  libavdevice    60.  1.100 / 60.  1.100
  libavfilter     9.  3.100 /  9.  3.100
  libswscale      7.  1.100 /  7.  1.100
  libswresample   4. 10.100 /  4. 10.100
  libpostproc    57.  1.100 / 57.  1.100
[mjpeg @ 0x56124978fec0] Format mjpeg detected only with low score of 25, misdetection possible!
Input #0, mjpeg, from 'fd:':
  Duration: N/A, bitrate: N/A
  Stream #0:0: Video: mjpeg (Baseline), yuvj422p(pc, bt470bg/unknown/unknown), 1056x704, 25 tbr, 1200k tbn
Stream mapping:
  Stream #0:0 -> #0:0 (mjpeg (native) -> rawvideo (native))
[swscaler @ 0x5612497fc940] deprecated pixel format used, make sure you did set range correctly
    Last message repeated 3 times
[video4linux2,v4l2 @ 0x561249792240] ioctl(VIDIOC_G_FMT): Invalid argument
[out#0/video4linux2,v4l2 @ 0x561249792e40] Could not write header (incorrect codec parameters ?): Invalid argument
[vost#0:0/rawvideo @ 0x561249794380] Error initializing output stream:
Conversion failed!

`

**Comment by weebney** [comment]
1. Try rebooting. Seems like v4l2loopback might be stuck "in use" which is a very common problem with the module.
2. I pushed another new version just now that automatically attempts to fix this when it happens.

**Comment by weebney** [comment]
Going to mark this closed as it's a known issue; see #6
